src/plugins/dataflow/model/utilities/node.ts

```diff
@@ -317,7 +319,7 @@ export const NodeSensorTypes = [
   {
     name: "Surface Pressure",
     type: "fsr-reading",
-    units: "f(n)",
+    units: "g",
```

Comment:
Just double checking if this is correct, since 'g' is not a typical unit for pressure

Reply:
ya.... I might have expected N or ftlb or something?

Reply:
Interesting - I got the g when I looked up the [specs on the FSR itself](https://www.sparkfun.com/datasheets/Sensors/Pressure/fsrguide.pdf) and found "g".  

Revisiting the question for this, I found [a bit of a qualifier on the adafruit site](https://learn.adafruit.com/force-sensitive-resistor-fsr/overview#how-to-measure-force-slash-pressure-with-an-fsr-365211), saying that it really could be understood as "N * 100" and not "g".  

[This article](https://lastminuteengineers.com/fsr-arduino-tutorial/) seems to say something similar, using "N" and "lbs".  So we could say "N" or maybe f(N)? @lbondaryk I think you and Aaron would be the most qualified to say.

Reply:
You can see that I originally put "f(n)", because a thought that said something like "some function of the original thing, which is newtons".

Reply:
So, is it really pressure that the sensor is reading - in which case you'd want Pascals?  Or force (Newtons)?  If it's really reporting (as I suspect) "I'm making contact with a thing and it's pushing back" then converting to N seems great - and perhaps renaming the item in the display to say force rather than pressure.

it seems like you must have encountered this question before, since sensors can send readings on very arbitrary scales, of whether you display those readings as-is (in some weird units that the user has to deal with) or convert them into standard units before display.

Reply:
> it seems like you must have encountered this question before

Yes - and that's how I ended up at f(n), which should have been "f(N)".  

With regards to calling it "Force" - or "Pressure" there is a degree to which surface area applied does in fact affect the reading, but I'm not enough of an engineer to understand exactly how much (pages 5, 6, 7) [here](https://www.sparkfun.com/datasheets/Sensors/Pressure/fsrguide.pdf). 

So, if it were up to me, I'd leave it as "f(N)" since that to my eyes is the most accurate representation we can make simple.  But, @lbondaryk , please do weigh in and I've also posted the question to Ido and Aaron.

Reply:
It is complicated by the fact that people using the sensors regularly interchange "force" and "pressure", since we often use the word "pressure" when talking about applying force with our hands.

Reply:
Thanks, @bgoldowsky for bringing this up, after going over spec sheet with Leslie we have arrived at `psi`.